index.html

```diff
@@ -243,220 +305,211 @@
+                                <a href="https://community.mozilla.org/groups/mozneurons" target="_blank">
+                                <div class="card4" style="width: 250px; height: 210px;">
+                                    <p style="margin-left:10px;"><img class="img" src="./assets/MozNeurons.jpeg" alt="user"
+                                            height="90px" width="230px"></p>
+                                    <p style="font-size: 30px; margin-left: 8px;"><b>MozNeurons</b><br><span
+                                            style="font-size: 25px;">SVIT</span></p>
+                                </div>
+                                </a>
                             </div>
-                            </a>
                         </div>
-                    </div>
-                    <div class="col-lg-3 col-md-3 col-sm-12">
-                        <div class="card1" style="width: 250px; height: 210px; background-color: #FFAB0F;">
+                        <div class="col-lg-3 col-md-3 col-sm-12">
+                            <div class="card1" style="width: 250px; height: 210px; background-color: #FFAB0F;">
+                                <a href="" target="_blank">
+                                <div class="card4" style="width: 250px; height: 210px;">
+                                <p style="margin-left:10px;"><img class="img" src="./assets/mozguj-horizontal-black.png"
+                                        alt="user" height="100px" width="230px"></p>
+                                <p style="font-size: 30px; margin-left: 8px;"><b>MozPU</b><br><span
+                                        style="font-size: 25px;">PU</span></p>
+                                </div>
+                                </a>
+                            </div>
+                        </div>
+                    
+                        <div class="col-lg-3 col-md-3 col-sm-12">
+                        <div class="card1" style="width: 250px; height: 210px; background-color: #00A2FF;">
                             <a href="" target="_blank">
                             <div class="card4" style="width: 250px; height: 210px;">
-                            <p style="margin-left:10px;"><img class="img" src="./assets/mozguj-horizontal-black.png"
-                                    alt="user" height="100px" width="230px"></p>
-                            <p style="font-size: 30px; margin-left: 8px;"><b>MozPU</b><br><span
-                                    style="font-size: 25px;">PU</span></p>
+                                <p style="margin-left:10px;"><img class="img" src="./assets/mozguj-horizontal-black.png"
+                                        alt="user" height="100px" width="230px"></p>
+                                <p style="font-size: 30px; margin-left: 8px;"><b>MozNUV</b><br><span
+                                        style="font-size: 25px;">NUV</span></p>
```

Comment:
Why the change from PU to NUV?

Reply:
Oops, sorry might have done that by mistake because in my local I had two PU and no NUV so that's why. Sorry for that.